Cakephp 2.7's FlashComponent requires flash elements to be in subdir Elements/Flash.

huhwatnou opened the pull request:
Needed to make e.g. the AuthComponent example with Cakephp 2.7

huhwatnou committed:
Cakephp 2.7's FlashComponent requires flash elements to be in subdir Elements/Flash.

wvdongen commented:
@slywalker please merge this

wvdongen commented:
One small addition to this merge request. In order for this to properly work with Cake 2.7 the following should be added to the top of `alert.ctp`:

``` php
if (isset($params['class'])) {
    $class = $params['class'];
}
```

Usage example in Cake 2.7:

``` php
$this->Flash->alert('Alert message', [
  'plugin' => 'BoostCake',
  'params' => ['class' => 'alert-success'],
]);
```

kim-dongit committed:
Allow use of the class parameter.

kim-dongit committed:
Use FlashComponent::alert() in examples.

Use FlashComponent::alert() in examples as SessionComponent::setFlash() is deprecated since cakephp 2.7 and the SessionComponent does not exist since cakephp 3.0.0.

huhwatnou committed:
Merge pull request #1 from kim-dongit/master

Use FlashComponent

huhwatnou commented:
Hi Slywalker, would you please merge these changes so that the plugin can be used with CakePHP >=3.0.0 and safely used with >=2.7?
